### PR TITLE
Fixes #17501 - Display tooltips on dashboard widgets

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

File: webpack/assets/javascripts/foreman_tools.js [hunk 1]
```diff
@@ -43,3 +43,16 @@ export function userInheritedRolesUpdater(id) {
     $(this).parents('.dropdown').find('.btn').html($(this).text() + ' <span class="caret"></span>');
   });
 }
+
+export function activateTooltips(el = $('body')) {
```
Comment (ohadlevy) [inline webpack/assets/javascripts/foreman_tools.js:47]: could you please add tests to this function?

**Comment by tbrisker** [comment]
@ohadlevy and now with tests

File: webpack/assets/javascripts/foreman_tools.js [hunk 2]
```diff
@@ -43,3 +43,16 @@ export function userInheritedRolesUpdater(id) {
...
+  el.find('[rel="twipsy"]').tooltip({ container: 'body' });
+  el.find('.ellipsis').tooltip({ container: 'body',
+                                 title: function () {
```
Comment (ohadlevy) [inline webpack/assets/javascripts/foreman_tools.js]: nitpick: you can use write something like:
```js
const title = () => {
  return (this.scrollWidth > this.clientWidth ? this.textContent : null);
 };


```

Comment (tbrisker) [reply webpack/assets/javascripts/foreman_tools.js]: right, still not used to all the nice es6 features :) pushing now fixed

**Comment by ohadlevy** [comment]
thanks @tbrisker

**Pull request merged**